pkg/loki/modules.go

```diff
@@ -96,8 +97,23 @@ const (
 	UsageReport              string = "usage-report"
 )
 
+func (t *Loki) initRegisterer() prometheus.Registerer {
```

Comment:
nit: The name of this function makes it look like there is a module called Registerer.

Reply:
That's true, I think I'll leave this for now and refactor it when I get rid of the reliance on the `prometheus.DefaultXXX` references